packages/optimizers/terser/src/TerserOptimizer.js

```diff
@@ -25,6 +25,7 @@ export default new Optimizer({
       ['.terserrc', '.uglifyrc', '.uglifyrc.js', '.terserrc.js'],
     );
 
+    let originalMap = map ? await map.stringify({}) : null;
```

Comment:
I was thinking of changing #4281 to just save vlq mappings in cache and letting transformers handle everything. (instead of using flatbuffers)

This should improve performance in some cases and probably make it more portable towards the repl. Not sure yet, will have to run some experiments and benchmarks first.